data/tests/Benchmark 3 - 120k #3/definitions/ChapterBreak/47583370_7.def.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
It seems that ChapterBreak has the very same samples than the `#1` version. There should be at least 4 samples reviewed, and we might need to review another two. These are of especial relevance, since they show up at the very beginning.